apps/betterangels-backend/clients/schema.py

```diff
@@ -130,46 +192,68 @@ def _validate_hmis_profiles(hmis_profiles: list[dict[str, Any]]) -> list[dict[st
             )
             .exists()
         ):
-            errors.append(_build_error("hmisProfiles", f"{idx}__hmisId", ErrorMessageEnum.HMIS_ID_IN_USE.name))
+            errors.append(
+                {
+                    "field": "hmisProfiles",
+                    "location": f"{idx}__hmisId",
+                    "errorCode": ErrorCodeEnum.HMIS_ID_IN_USE.name,
+                }
+            )
 
     return errors
 
 
-def _validate_contacts(contacts: list[dict[str, Any]]) -> list[dict[str, Any]]:
+def validate_contacts(contacts: list[dict[str, Any]]) -> list[dict[str, Any]]:
     errors = []
 
     for idx, contact in enumerate(contacts):
+        if not contact.get("phone_number"):
+            continue
+
         try:
-            phonenumber_field.validators.validate_international_phonenumber(contact["phone_number"])
+            validate_international_phonenumber(contact["phone_number"])
         except ValidationError:
-            errors.append(_build_error("contacts", f"{idx}__phoneNumber", ErrorMessageEnum.INVALID_PHONE_NUMBER.name))
+            errors.append(
+                {
+                    "field": "contacts",
+                    "location": f"{idx}__phoneNumber",
+                    "errorCode": ErrorCodeEnum.PHONE_NUMBER_INVALID.name,
+                }
+            )
 
     return errors
 
 
-def _validate_client_profile_data(data: dict) -> None:
+def validate_client_profile_data(data: dict) -> dict[Any, Any]:
     """Validates the data for creating or updating a client profile."""
-    errors = []
+    errors: list = []
+
+    user = None
+
+    if value_is_set(data.get("id")):
+        user = User.objects.filter(client_profile__id=data["id"]).first()
```

Comment:
```suggestion
        user = User.objects.get(client_profile__id=data["id"])
```
and can wrap that in a try block if you want a cuter error but figure it should fail explicitly if an id that's not tied to a user is passed